src/Entity/PageManager.php

```diff
@@ -35,10 +36,18 @@ final class PageManager extends BaseEntityManager implements PageManagerInterfac
     public function __construct(
         string $class,
         ManagerRegistry $registry,
-        private SlugifyInterface $slugify,
+        private SlugifyInterface|SluggerInterface $slugger, // NEXT_MAJOR: Reduce $slugger to SluggerInterface
         private array $defaults = [],
         private array $pageDefaults = [],
     ) {
+        if ($slugger instanceof SlugifyInterface) {
+            @trigger_error(\sprintf(
+                '%s is deprecated since version 4.11.0 and will be removed in 5.0, use %s instead of.',
```

Comment:
Maybe we can be more specific? Like 

```
Passing an instance of ... to ... is deprecated since ... . Use an instance of ... instead.

Reply:
```diff
+ Passing an instance of ... to ... is deprecated.
- since ... . 
+Use an instance of ... instead.
```

I keep the message until `deprecated` because `symfony deprecation` add this into the message.